### ISSUE TITLE
[Idea] Support explicit installation of package from the registry

### ISSUE THREAD

**Issue by jsoriano** [elastic/elastic-package#1591]
Add support in `elastic-package install` to install a package from the registry without requiring a working copy of the package.

It would require to add some flag for this. An option would be to do the same as in benchmarks, with the `--package-from-registry` flag, that would receive a `<package>-<version>` and would use it to install it from the registry, maybe we could also support `--package-from-registry <package>`, that would install the latest version available in the registry.
Another option could be to add two flags, `--package` and `--version`, but this would be less explicit and we would need to check how it behaves with other flags.

The use case would be to prepare environments while testing other packages or other individual assets.

Request comes from a discussion with @philippkahr and @ruflin.

**Comment by philippkahr** [comment]
Can you define the package registry with your own hosted EPR or will it only pull from epr.elastic.co?

**Comment by jsoriano** [comment]
> Can you define the package registry with your own hosted EPR or will it only pull from epr.elastic.co?

If we want to support installing from custom package registries we would probably need to download the zip from this registry, and then install the zip. This is indeed a possible option.

Is this something you would need?

**Comment by philippkahr** [comment]
definitely, in a lot of customer environments they are airgapped and don't have access to epr.elastic.co. Therefore either specifying the url using an environment variable like: `package_registry_url: my.custom.epr.com` should do the trick? then we can leave the flag with `--package-from-registry`?

**Comment by jsoriano** [comment]
Yep, makes sense, added a line about this in the description.

**Comment by kpollich** [comment]
We haven't revisited this issue in a while, so we're closing it due to inactivity. Please reopen with a comment if you think this is a mistake. Thanks!
